apps/google-analytics-4/shared/types/ReportResponse.model.ts

```diff
@@ -0,0 +1,40 @@
+export interface RunReportResponse {
```

Comment:
I'm not familiar with a `model.ts` file (though I can tell the naming doesn't actually do anything here). Googling suggests some angular results, but nothing clear. Is this a convention we'd like to work with for our interface files going forward?

Reply:
I'm certainly fine with it, just wanted to pick up some learning for myself